ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/IntTopkDistinctKudafTest.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.function.udaf.topkdistinct;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class IntTopkDistinctKudafTest {
+
+  Integer[] valueArray;
+  @Before
+  public void setup() {
+    valueArray = new Integer[]{10, 30, 45, 10, 50, 60, 20, 60, 80, 35, 25,
+                              60, 80};
+
+  }
+
+  @Test
+  public void shouldAggregateTopK() {
+    TopkDistinctKudaf<Integer> doubleTopkDistinctKudaf = new TopkDistinctKudaf(0, 3, Integer.class);
```

Comment:
A variable name `doubleTopkDistinctKudaf` is misleading in all these cases.

Reply:
Corrected the variable name.